07-IO Streams/IOStreams/TestTasks.cs

```diff
@@ -34,7 +34,30 @@ public static IEnumerable<PlanetInfo> ReadPlanetInfoFromXlsx(string xlsxFileName
 			//         /xl/sharedStrings.xml      - dictionary of all string values
 			//         /xl/worksheets/sheet1.xml  - main worksheet
 
-			throw new NotImplementedException();
+
+			using (var pac = Package.Open(xlsxFileName))
+			{
+				var uri1 = new Uri("/xl/worksheets/sheet1.xml", UriKind.Relative);
+				var uri2 = new Uri("/xl/sharedStrings.xml", UriKind.Relative);
+
+				var part = pac.GetPart(uri1);
+				var part2 = pac.GetPart(uri2);
+
+				var docNamesPlanet = XDocument.Load(part2.GetStream());
```

Comment:
don't forget to despose of streams

Reply:
also in English noun modifiers go before the noun:

-  variable contains the doc that contains names => `names` is a modifier for the `doc` => namesDoc.

- names of the planet => `planet` is a modifier for the `names`,

So the more readable name is planetNamesDoc - the doc that contains the names of the planets

Reply:
https://learnenglish.britishcouncil.org/en/english-grammar/adjectives/noun-modifiers